input/fsh/codeSystem/codeSystem.fsh

```diff
@@ -0,0 +1,1319 @@
+* #9860129 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860129 ^designation[+].language = #ht
+* #9860129 ^designation[=].value = "Wi"
+* #9860129 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860129 ^designation[+].language = #ht
+* #9860129 ^designation[=].value = "Vre"
+* #9860129 ^designation[+].language = #ur
+* #9860129 ^designation[=].value = "ہاں"
+* #9860129 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860129 ^designation[+].language = #km
+* #9860129 ^designation[=].value = "បាទ"
+* #9860129 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860129 ^designation[+].language = #km
+* #9860129 ^designation[=].value = "ចាស"
+* #9860129 ^designation[+].language = #ne
+* #9860129 ^designation[=].value = "हुन्छ"
+* #9860129 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860129 ^property[0].code = #conceptclass
+* #9860129 ^property[=].valueString = "Misc"
+* #9860129 ^property[+].code = #datatype
+* #9860129 ^property[=].valueString = "N/A"
+* #9860131 "New client"
+* #9860131 ^designation[0].language = #en
+* #9860131 ^designation[=].value = "New client"
+* #9860131 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860131 ^designation[+].language = #en
+* #9860131 ^designation[=].value = "New patient"
+* #9860131 ^property[0].code = #conceptclass
+* #9860131 ^property[=].valueString = "Question"
+* #9860131 ^property[+].code = #datatype
+* #9860131 ^property[=].valueString = "Coded"
+* #9860133 "Transfer in date"
+* #9860133 ^designation[0].language = #en
+* #9860133 ^designation[=].value = "Transfer in date"
+* #9860133 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860133 ^property[0].code = #conceptclass
+* #9860133 ^property[=].valueString = "Question"
+* #9860133 ^property[+].code = #datatype
+* #9860133 ^property[=].valueString = "Date"
+* #9860135 "Transit patient" "A patient who is being seen outside of their regular care setting temporarily to receive medication refills, etc."
+* #9860135 ^designation[0].language = #en
+* #9860135 ^designation[=].value = "Transient patient"
+* #9860135 ^designation[+].language = #en
+* #9860135 ^designation[=].value = "Transit patient"
+* #9860135 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860135 ^property[0].code = #conceptclass
+* #9860135 ^property[=].valueString = "Misc"
+* #9860135 ^property[+].code = #datatype
+* #9860135 ^property[=].valueString = "N/A"
+* #9860137 "re-enroll"
+* #9860137 ^designation[0].language = #en
+* #9860137 ^designation[=].value = "re-enroll"
+* #9860137 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860137 ^property[0].code = #conceptclass
+* #9860137 ^property[=].valueString = "Misc"
+* #9860137 ^property[+].code = #datatype
+* #9860137 ^property[=].valueString = "N/A"
+* #discontinuation_date "Discontinuation date"
+* #discontinuation_date ^designation[0].language = #en
+* #discontinuation_date ^designation[=].value = "Discontinuation date"
+* #discontinuation_date ^designation[=].use.code = #Fully-Specified
```

Comment:
The value for Fully Specified is different in this row ("Fully-Specified") in comparison to all other rows, which say "FULLY_SPECIFIED"

Reply:
@jamlung-ri , Not able to figureout what concept we can use from CIEL that corresponds to this. You can help me find an alternative for this.

Reply:
@jamlung-ri , I have removed this and replace with "HIV program end date" #164094 as you suggested on the slack channel.